helm/fence/values.yaml

```diff
@@ -112,11 +112,11 @@ replicaCount: 1
 
 image:
   # -- (string) The Docker image repository for the fence service
-  repository: quay.io/cdis/fence
+  repository: quay.io/ohsu-comp-bio/fence
   # -- (string) When to pull the image. This value should be "Always" to ensure the latest image is used.
   pullPolicy: Always
   # -- (string) Overrides the image tag whose default is the chart appVersion.
-  tag: "master"
+  tag: "feat_multi_part_bucket_param"
```

Comment:
fence in secrets doesn't currently have any specific image specified. Maybe that should be added instead of changing the helm chart to `feat_multi_part_bucket_param`